polaris-core/src/main/java/org/apache/polaris/core/PolarisConfiguration.java

```diff
@@ -191,4 +191,12 @@ public static <T> Builder<T> builder() {
               "If set to true, allows tables to be dropped with the purge parameter set to true.")
           .defaultValue(true)
           .build();
+
+  public static final PolarisConfiguration<Boolean> SUPPORT_WASB_CATALOG =
+      PolarisConfiguration.<Boolean>builder()
+          .key("SUPPORT_WASB_CATALOG")
+          .description(
+              "If set to true, allows the creation of catalogs with storage locations using Azure WASB prefix.")
+          .defaultValue(false)
```

Comment:
Considering we support `FILE` by _default_, I don't think we need to be so strict so as to not support WASB by default.

Reply:
I think `FILE` is different, as it works out of the box. `wasb` does not. In my mind, opting in to something that doesn't work quite right is different from opting out of something you don't intend to work.

Reply:
Isn't it the case that WASB does work out of the box for Polaris just like `FILE`, but the concern is that some clients might not be able to use credential vending with WASB (until the next Iceberg release)? Clients also can't use credential vending with `FILE`.